modern/src/settings/GroupsPage.js

```diff
@@ -45,6 +45,7 @@ const GroupsPage = () => {
   const actionCommand = {
     key: 'command',
     title: t('deviceCommand'),
+    tooltip: t('commandSendTitle'),
```

Comment:
Why can't we just use title? Then we don't need extra strings either.

Reply:
I wanted to use the specific "Send Command" instruction, but I could just use the title and leave it as "Command"

Reply:
Let's leave it as "command".